app/helpers/admin/visit_requests_helper.rb

```diff
@@ -0,0 +1,17 @@
+module Admin
+  module VisitRequestsHelper
+
+    STATUS_CLASS = {
+        ::VisitRequest::PENDING => 'bg-info',
+        ::VisitRequest::APPROVED => 'bg-success',
+        ::VisitRequest::CANCELED => 'bg-danger'
+    }
+
+    def status_cell(status)
+      content_tag :td, class: STATUS_CLASS[status.to_sym] do
```

Comment:
@sveredyuk I am not using helper modules to extract view logic, so show me the path if this is wrong way